spec/stack.js

```diff
@@ -19,4 +19,37 @@ describe('Stack', () => {
         .to.alter(() => myStack.length(), { from: 0, to: 1 })
     })
   })
+
+  context('pop()', () => {
+    it('returns and removes the top element in a stack or null if stack is empty', () => {
+      const myStack = new Stack()
+
+      myStack.push('popcorn')
+      expect(() => myStack.pop())
+        .to.alter(() => myStack.length(), { from: 1, to: 0 })
+    })
+  })
+
+  context('peek()', () => {
+  it('returns the top element in the stack or null if the stack is empty', () => {
```

Comment:
The test description indicates two tests again - consider splitting this into two discrete tests that tests each of these assertions.  As well, this test is not testing for a `null` return value in the event that the stack is empty.